misc/src/androidTest/java/com/example/snippets/DeviceCompatibilityModeTestJavaSnippets.java

```diff
@@ -33,16 +34,18 @@ public void activity_launched_notLetterBoxed() {
         try (ActivityScenario<MainActivity> scenario =
             ActivityScenario.launch(MainActivity.class)) {
                 scenario.onActivity( activity -> {
-                    assertFalse(isLetterboxed(activity));
+                    assertFalse(activity.isLetterboxed(activity));
```

Comment:
There is no parameter now to this method, so we can remove the `activity` being passed here.

```suggestion
                    assertFalse(activity.isLetterboxed());
```